### PR TITLE
You can now recolo(u)r lightbulbs (#44122)

### PR DESCRIPTION
**Pull request by Cenrus**
https://github.com/tgstation/tgstation/pull/44122

Use a spraycan on lights to recolour them. Nice if you want to create an atmospheric bar

:cl: nemvar
add: Colored lights now shine in different colours.
/:cl:

### CODE REVIEW

File: code/modules/power/lighting.dm [hunk 1]
```diff
@@ -401,6 +403,10 @@
 			removeStaticPower(static_power_used, AREA_USAGE_STATIC_LIGHT)
 
 	broken_sparks(start_only=TRUE)
+	
+/obj/machinery/light/update_atom_colour()
+	..()
+	update()
 
 /obj/machinery/light/update_atom_colour()
```
Comment (ike709) [inline code/modules/power/lighting.dm]: hmmm

**Comment by ike709** [comment]
How does this interact with bacon's area-based lighting?

**Comment by Cenrus** [comment]
image

Works fine

**Comment by eeSPee** [comment]
But can it change the coulours of the light? I think not.

**Comment by Sinmero** [comment]
Works fine.

File: code/modules/power/lighting.dm (hunk 1 above)
Comment (PowerfulBacon) [reply code/modules/power/lighting.dm]: There are 2 update atom colour procs btw

File: code/modules/power/lighting.dm [hunk 2]
```diff
@@ -401,6 +403,10 @@
 			removeStaticPower(static_power_used, AREA_USAGE_STATIC_LIGHT)
 
 	broken_sparks(start_only=TRUE)
+	
+/obj/machinery/light/update_atom_colour()
+	..()
+	update()
```
Comment (PowerfulBacon) [inline code/modules/power/lighting.dm]:
```suggestion
```

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]

**Comment by Cenrus** [comment]
I could have sworn I already got rid of that extra proc, sorry

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**